### PR TITLE
data:shubham185y

### PR DESCRIPTION
**Pull request by Shubham185y**

### CODE REVIEW

**Review by accodes21 — CHANGES_REQUESTED** [review]
Change your filename to that of your github username i.e ```Shubham185y.json```

File: Shubham185y.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Shubham Yadav",
+  "type": "personal",
+  "bio": "Software Devloper",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Shubham185y",
```
[the remark this answers is not in the document]
Comment (vedprakashnautiyal) [reply Shubham185y.json:8]: This account doesn't exist.......!!??

**Comment by vedprakashnautiyal** [comment]
Also your json file should be in the data folder......

File: Shubham185y.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shubham185y) [reply Shubham185y.json]: thank you i miss placed y in the username a bit confusing from github and twitter

**Comment by SaraJaoude** [comment]
> > Also your json file should be in the data folder......
> 
> how to add it to data folder

Please take a look at the QuickStart guide in Docs as this should help you: https://linkfree.eddiehub.io/docs/quickstart

**Review by khushalarora — APPROVED** [review]
Thank you for contributing  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**